utils/sort-glossary.py

```diff
@@ -0,0 +1,335 @@
+    ('it', 'Italian'),
+    ('jv', 'Javanese'),
+    ('ja', 'Japanese'),
+    ('kl', 'Kalaallisut; Greenlandic'),
+    ('kn', 'Kannada'),
+    ('ks', 'Kashmiri'),
+    ('ka', 'Georgian'),
+    ('kr', 'Kanuri'),
+    ('kk', 'Kazakh'),
+    ('km', 'Central Khmer'),
+    ('ki', 'Kikuyu; Gikuyu'),
+    ('rw', 'Kinyarwanda'),
+    ('ky', 'Kirghiz; Kyrgyz'),
+    ('kv', 'Komi'),
+    ('kg', 'Kongo'),
+    ('ko', 'Korean'),
+    ('kj', 'Kuanyama; Kwanyama'),
+    ('ku', 'Kurdish'),
+    ('lo', 'Lao'),
+    ('la', 'Latin'),
+    ('lv', 'Latvian'),
+    ('li', 'Limburgan; Limburger; Limburgish'),
+    ('ln', 'Lingala'),
+    ('lt', 'Lithuanian'),
+    ('lb', 'Luxembourgish; Letzeburgesch'),
+    ('lu', 'Luba-Katanga'),
+    ('lg', 'Ganda'),
+    ('mk', 'Macedonian'),
+    ('mh', 'Marshallese'),
+    ('ml', 'Malayalam'),
+    ('mi', 'Maori'),
+    ('mr', 'Marathi'),
+    ('ms', 'Malay'),
+    ('Mi', 'Micmac'),
+    ('mk', 'Macedonian'),
+    ('mg', 'Malagasy'),
+    ('mt', 'Maltese'),
+    ('mn', 'Mongolian'),
+    ('mi', 'Maori'),
+    ('ms', 'Malay'),
+    ('my', 'Burmese'),
+    ('na', 'Nauru'),
+    ('nv', 'Navajo; Navaho'),
+    ('nr', 'Ndebele, South; South Ndebele'),
+    ('nd', 'Ndebele, North; North Ndebele'),
+    ('ng', 'Ndonga'),
+    ('ne', 'Nepali'),
+    ('nl', 'Dutch; Flemish'),
+    ('nn', 'Norwegian Nynorsk; Nynorsk, Norwegian'),
+    ('nb', 'Bokmål, Norwegian; Norwegian Bokmål'),
+    ('no', 'Norwegian'),
+    ('oc', 'Occitan (post 1500)'),
+    ('oj', 'Ojibwa'),
+    ('or', 'Oriya'),
+    ('om', 'Oromo'),
+    ('os', 'Ossetian; Ossetic'),
+    ('pa', 'Panjabi; Punjabi'),
+    ('fa', 'Persian'),
+    ('pi', 'Pali'),
+    ('pl', 'Polish'),
+    ('pt', 'Portuguese'),
+    ('ps', 'Pushto; Pashto'),
+    ('qu', 'Quechua'),
+    ('rm', 'Romansh'),
+    ('ro', 'Romanian; Moldavian; Moldovan'),
+    ('ro', 'Romanian; Moldavian; Moldovan'),
+    ('rn', 'Rundi'),
+    ('ru', 'Russian'),
+    ('sg', 'Sango'),
+    ('sa', 'Sanskrit'),
+    ('si', 'Sinhala; Sinhalese'),
+    ('sk', 'Slovak'),
+    ('sk', 'Slovak'),
+    ('sl', 'Slovenian'),
+    ('se', 'Northern Sami'),
+    ('sm', 'Samoan'),
+    ('sn', 'Shona'),
+    ('sd', 'Sindhi'),
+    ('so', 'Somali'),
+    ('st', 'Sotho, Southern'),
+    ('es', 'Spanish; Castilian'),
+    ('sq', 'Albanian'),
+    ('sc', 'Sardinian'),
+    ('sr', 'Serbian'),
+    ('ss', 'Swati'),
+    ('su', 'Sundanese'),
+    ('sw', 'Swahili'),
+    ('sv', 'Swedish'),
+    ('ty', 'Tahitian'),
+    ('ta', 'Tamil'),
+    ('tt', 'Tatar'),
+    ('te', 'Telugu'),
+    ('tg', 'Tajik'),
+    ('tl', 'Tagalog'),
+    ('th', 'Thai'),
+    ('bo', 'Tibetan'),
+    ('ti', 'Tigrinya'),
+    ('to', 'Tonga (Tonga Islands)'),
+    ('tn', 'Tswana'),
+    ('ts', 'Tsonga'),
+    ('tk', 'Turkmen'),
+    ('tr', 'Turkish'),
+    ('tw', 'Twi'),
+    ('ug', 'Uighur; Uyghur'),
+    ('uk', 'Ukrainian'),
+    ('ur', 'Urdu'),
+    ('uz', 'Uzbek'),
+    ('ve', 'Venda'),
+    ('vi', 'Vietnamese'),
+    ('vo', 'Volapük'),
+    ('cy', 'Welsh'),
```

Comment:
@froggleston Welsh is already present above in line 47. Maybe sort items so that we can easier spot accidental duplicates?

Reply:
Great spot! I've removed the duplicates.